Car.java

```diff
@@ -0,0 +1,16 @@
+package com.mycompany.carcrash;
+
+import com.mycompany.carcrash.CustomExceptions.CarAccidentException;
+
+public class Car {
+    public void driveCar() throws Exception
+    {
```

Comment:
Curly brackets(formatting) should be consistent in all files that is starting curly bracket either in same line or next line.